packages/react-core/src/Authenticator/hooks/useAuthenticator/__tests__/useAuthenticator.spec.tsx

```diff
@@ -69,37 +72,50 @@ describe('useAuthenticator', () => {
     expect(result.error?.message).toBe(USE_AUTHENTICATOR_ERROR);
   });
 
-  it('returns the expected values', () => {
-    const { result } = renderHook(useAuthenticator, {
+  it('returns the expected values', async () => {
+    const { result, waitForNextUpdate } = renderHook(useAuthenticator, {
       wrapper: Wrapper,
     });
 
+    await waitForNextUpdate();
```

Comment:
Added this (and the other calls to `waitForNextUpdate` in this file) to clean up warnings in the console around async actions and effects